pokedex/docs/doc.typ

```diff
@@ -431,8 +440,8 @@ Register (CSR) to models.
 All CSRs operations are handled by two main APIs that are generated by the
 `codegen` CLI tool.
 
-- Read API: `func ReadCSR(csr_number: bits(12)) => bits(xlen)`
-- Write API: `func WriteCSR(csr_number: bits(12), value: bits(xlen))`
+- Read API: `func ReadCSR(csr_number: bits(12)) => bits(32)`
```

Comment:
Reading CSR may raise exception depends on arch state. e.g. Whether reading performance counter may succeed depend on m/scounteren. Suggest change the return type to Result.